pkg/apis/minio.min.io/v1/types.go

```diff
@@ -142,9 +142,15 @@ type ExposeServices struct {
 	// MinIO tells operator whether to expose the MinIO service
 	// +optional
 	MinIO bool `json:"minio,omitempty"`
+	// If provided,apply these annotations to MinIO ClusterIP service
+	// +optional
+	MinIOAnnotations map[string]string `json:"minioAnnotations,omitempty"`
```

Comment:
this makes me wonder if we should introduce a subtype instead because if annotations are needed, perhaps labels will be needed later as well as other metadata fields, should we do it in the same PR?

Reply:
Makes sense, added support for metadata as well in another subtype. PTAL